test/unit/au/org/emii/aatams/OpenIntervalTests.groovy

```diff
@@ -0,0 +1,20 @@
+package au.org.emii.aatams
+
+import org.joda.time.*
+import grails.test.*
+
+class OpenIntervalTests extends GrailsUnitTestCase {
+    def now = new DateTime()
+
+    void testOverlaps() {
+        def openInterval = new OpenInterval(now)
+        assertFalse(openInterval.overlaps(new Interval(now.minusDays(2), now.minusDays(1))))
+        assertTrue(openInterval.overlaps(new Interval(now.plusDays(1), now.plusDays(2))))
```

Comment:
Just for completeness it might be nice to have a test where the interval starts before the open interval but finishes within it.